js/join.js

```diff
@@ -49,12 +49,270 @@ function inputEventAdder(field) {
   });
 }
 
+const countryList = [
```

Comment:
This should be in a constants file

Reply:
Sorry but this repository does not have any constants file. So I did not create one.

Reply:
Can you create one?

Reply:
Ok will do

Reply:
Done this in the new Push

Reply:
done